toolchain/semantics/semantics_ir.h

```diff
@@ -331,7 +331,7 @@ class SemanticsIR {
 
   // The type of the empty tuple. This is special-cased due to its use in
   // implicit function returns.
-  SemanticsTypeId empty_tuple_type_id_ = SemanticsTypeId::Invalid;
+  SemanticsTypeId empty_tuple_type_id_ = SemanticsTypeId::TypeType;
```

Comment:
This looks incorrect; what's the type of the empty tuple? i.e., `() as type`

Note, you might want to just remove this, but it definitely needs to be correct if it remains.